Gemfile

```diff
@@ -59,6 +59,7 @@ gem "compass-rails",  "2.0.5"
 gem "sass-rails",     "5.0.4"
 gem "autoprefixer-rails", "6.3.6.2"
 gem "bootstrap-switch-rails", "3.3.3"
+gem "font-awesome-rails", "4.4.0.0"
```

Comment:
Note that, currently, it's only used on desktop, not mobile. Of course, my goal is to bring it to mobile too, because this is where MD is the most diffucult to use. Do you think the icons library size is going to be a big problem?